new-dti-website/components/team/MemberDisplay.tsx

```diff
@@ -0,0 +1,180 @@
+import { useRef, useState } from 'react';
+import MemberGroup from './MemberGroup';
+import Icon from '../icons';
+import FA23Members from '../../../backend/src/members-archive/fa23.json';
+
+const MemberDisplay: React.FC = () => {
+  const [selectedRole, setSelectedRole] = useState<string>('Full Team');
+  const [selectedMember, setSelectedMember] = useState<IdolMember | undefined>(undefined);
+
+  const memberDetailsRef = useRef<HTMLInputElement>(null);
+
+  const allMembers = FA23Members.members as IdolMember[];
+  const roles: {
+    [key in Role]: {
+      roleName: string;
+      description: string;
+      members: IdolMember[];
+      order: string[];
```

Comment:
Yeah, dividing the two member types would be good for privacy reasons. Members can then edit what they want to display on their profile via IDOL, but that's another feature we can talk about.

The team page has risen some concerns about the current database, which we should resolve first before deciding on using `NovaMember` (which I'm leaning towards). First, we need to keep track of the colleges of each member. Majors should be standardized (CS vs. Computer Science) -- these are important when determining the represented colleges and majors in the About portion of the page. Another issue is how to deal with multiple subroles within a main  role, akin to the advisor discussion on a different PR. That is, how to expand `RoleDescription` or some other field to include advisors, APM, PMM, different lead roles, etc.

Reply:
w.r.t. `NovaMember` vs ` IdolMember`, that's really just about whether to use the same data and fields bewteen idol and the dti website. The `NovaMember` type can of course be changed to fit whatever the needs of the new dti website are. 

w.r.t. profile stuff:
- The subroles situation i kinda touched on in #588. Related to #471 as well. I think that's the best way to handle this. 
- In terms of standardizing majors, there's a tradeoff that needs to be considered there. The IDOL profile editor is intended to allow self-servicing of changing things like your major. That's just a free-form text box to type your major in. The only "verification" happens at the stage when an idol admin has to check the diff before approving it. I'm not sure at that stage if rejecting a change is worth it because someone put "CS" instead of "Computer Science." The way to prevent that might be to use a dropdown with all the majors, but that requires maintaining a list of majors which is a pain. From a design perspective, I'm not sure where they stand on that, but from an engineering perspective, it might be fine imo as long as it's not something egregious like the major is misspelled or it's a fake major or something.